### DISCUSSION TITLE
Support for Proxmox

### DISCUSSION THREAD

**Discussion by zaxlofful** [opentofu/opentofu#416] [Q&A] [answered]
Is there support for the Proxmox Provider or planned? This is a big deal for the self-hosted community.

**Comment by cube2222** [comment, marked as answer]
Hey @ZaxLofful! All existing providers should work unchanged.

**Comment by cosmicvibes** [comment]
Hey all. I confirm it works fine. I just built the main branch, and it pulled the telmate/proxmox plugin and created my Proxmox vms same as it always did. Good job all! I will switch my binary to opentofu on my dev/lab setup so I can keep testing over the coming weeks. Cheers.
